go-backend/db/mock.go

```diff
@@ -39,3 +39,12 @@ func (m *DBMockStore) UpdateOrganization(ctx context.Context, org Organization,
 	args := m.Called(ctx, org, id)
 	return args.Get(0).(Organization), args.Error(1)
 }
+
+func (m *DBMockStore) GetUser(ctx context.Context, id int) (user User, err error) {
+	args := m.Called(ctx, id)
+	return args.Get(0).(User), args.Error(1)
+}
+func (m *DBMockStore) UpdateUser(ctx context.Context, usr User, id int) (updatedUser User, err error) {
```

Comment:
for being consistent, always call it `user`, not `usr`

Reply:
Okay I will do correct it.